articles/native-platforms/android/04-user-profile.md

```diff
@@ -0,0 +1,124 @@
+---
+title: User Profile
+description: This tutorial will show you how to use Lock10 to get the user's profile data.
+---
+
+This tutorial will show you how to use Lock10 to get the user's profile data in your android apps with Auth0.
+
+::: panel-info System Requirements
+This tutorial and seed project have been tested with the following:
+
+* AndroidStudio 2.0
+* Emulator - Nexus5X - Android 6.0 
+:::
+
+ <%= include('../../_includes/_github', {
```

Comment:
@tenmaster Hi, this file does not exist, please see [this PR](https://github.com/auth0/docs/pull/2119/files) and update the docs in this one accordingly. Thank you.

^ @chenkie